app/src/atoms/AtomButton.re

```diff
@@ -1,87 +1,100 @@
 module Styles = {
   open Css;
   open Particle;
+  open Util.CssHelper;
 
   let button =
-    merge([
-      style([
-        backgroundColor(Color.white()),
-        borderRadius(px(4)),
-        borderStyle(solid),
-        borderWidth(px(1)),
-        borderColor(Color.lightGray()),
-        color(Color.darkGray()),
-        cursor(`pointer),
-        height(px(48)),
-        outlineStyle(`none),
-        padding4(
-          ~top=px(13),
-          ~right=px(24),
-          ~bottom=px(14),
-          ~left=px(24),
-        ),
-        hover([backgroundColor(Color.almostWhite(~alpha=0.85, ()))]),
-        active([backgroundColor(Color.almostWhite(~alpha=0.2, ()))]),
-      ]),
-      style(Font.title),
-    ]);
+    style([
+      backgroundColor(Color.white()),
+      borderRadius(px(4)),
+      borderStyle(solid),
+      borderWidth(px(1)),
+      borderColor(Color.lightGray()),
+      color(Color.darkGray()),
+      cursor(`pointer),
+      height(px(48)),
+      outlineStyle(`none),
+      padding4(~top=px(13), ~right=px(24), ~bottom=px(14), ~left=px(24)),
+      hover([backgroundColor(Color.almostWhite(~alpha=0.85, ()))]),
+      active([backgroundColor(Color.almostWhite(~alpha=0.2, ()))]),
+    ])
+    << style(Font.title);
```

Comment:
Out of curiosity can you chain them? a la `style(...) << style(...) << style(...)`?

Reply:
Yes, my idea was that this would allow us to write more concise inline styles.